sourmash/commands.py

```diff
@@ -968,18 +968,18 @@ def gather(args):
 
         if not len(found):                # first result? print header.
             print_results("")
-            print_results("overlap     p_query p_match ")
-            print_results("---------   ------- --------")
+            print_results("overlap     p_query p_match avg_abund")
```

Comment:
can we get by with 'abund' or does that make things too cramped?

Reply:
(for the header line.)

Reply:
Happy either way-- I thought it was good to say avg? I imagine people will figure it out.